### ISSUE TITLE
Video doesn't open in PC after File.Move / File.Copy / Sending in to the Server

### ISSUE THREAD

**Issue by WarriorWork** [yasirkula/UnityNativeCamera#113]
Video doesn't open in PC after File.Move / File.Copy / Send in to the Server:
It occurs sometimes and is not persistent, most of the time it doesn't work alright.
I get errors like this one 
![image](https://github.com/yasirkula/UnityNativeCamera/assets/93075271/fd03f07e-7f09-4e94-a12c-1ee998f6cf17)

- Unity version: *e.g. 2018.3.1f1*
- Platform: Android / iOS
- Device: Xioami 12, iPad Air
- How did you download the plugin: through the Asset Store

**Comment by yasirkula** [comment]
Does the file have an extension?

**Comment by WarriorWork** [comment]
Yeah of course .mp4

> Does the file have an extension?

**Comment by yasirkula** [comment]
Can you play the video in VideoPlayer or via the example code's Handheld.PlayFullScreenMovie function on Android/iOS? This may be a codec issue. The video may be recorded in HEVC format. I'd try playing it with MPC-HC on PC since it supports lots of codecs.

**Comment by WarriorWork** [comment]
> Can you play the video in VideoPlayer or via the example code's Handheld.PlayFullScreenMovie function on Android/iOS? This may be a codec issue. The video may be recorded in HEVC format. I'd try playing it with MPC-HC on PC since it supports lots of codecs.

Yeah, it seems codecs issues! because when I run the video on the browser or via VLC and not via Media Player, it works!
So firstly thank you.
Additionally, if you have got any idea of how to solve it, Ill be happy to hear that...
The problem is when I transfer the video to the Server (via UnityWebRequest) it wouldn't open via Media Player :)
Thank you!

**Comment by WarriorWork** [comment]
> Can you play the video in VideoPlayer or via the example code's Handheld.PlayFullScreenMovie function on Android/iOS? This may be a codec issue. The video may be recorded in HEVC format. I'd try playing it with MPC-HC on PC since it supports lots of codecs.

Do you know if I can controll the codec by the NativeCamera plugin or in another way?
As I see the most popular codec is H.265... can I achieve that?

**Comment by yasirkula** [comment]
I couldn't find a way to explicitly specify the codec to use. As you've said, changing the quality may indirectly affect the codec. I'd expect this to be consistent on iOS devices but not on Android devices. I think you need either a native code on the device that converts the video from the recorded format to the desired format, or a code that does the same on the server.
